curriculum/challenges/english/blocks/workshop-html-video-player/68f18724d71db1dda8d6fec7.md

```diff
@@ -0,0 +1,44 @@
+---
+id: 68f18724d71db1dda8d6fec7
+title: Step 1
+challengeType: 0
+dashedName: step-1
+demoType: onLoad
+---
+
+# --description--
+
+Now you'll apply what you learned in previous lessons, you'll build a HTML Video Player.
+
+You have been provided the HTML5 boilerplate code.
+
+First, create an `h1` element and give it the text `Working with the HTML Video Element`.
+
+# --hints--
+
+Your `h1` element should contain the text `Working with the HTML Video Element`.
+
+```js
+const h1 = document.querySelector('h1');
+assert.strictEqual(h1?.textContent.trim(), 'Working with the HTML Video Element');
+```
```

Comment:
split this into two tests

one to check if they have an h1

another to check for the h1 text